common/models/models.go

```diff
@@ -141,3 +144,15 @@ type StreamSettings struct {
 	ScalingFactorAndroid int `json:"scaling_factor_android,omitempty" bson:"scaling_factor_android"`
 	ScalingFactoriOS     int `json:"scaling_factor_ios,omitempty" bson:"scaling_factor_ios"`
 }
+
+type Workspace struct {
+	ID          string `json:"id" bson:"_id,omitempty"`
+	Name        string `json:"name" bson:"name"`
+	Description string `json:"description" bson:"description"`
+	IsDefault   bool   `json:"is_default" bson:"is_default"`
+}
+
+// GenerateUUID generates a new UUID for the workspace
+func (w *Workspace) GenerateUUID() {
```

Comment:
I am not really sure we need this, Mongo automatically creates arbitrary `_id` when a document is added

Reply:
Changes in [commit](https://github.com/gifflet/GADS/commit/68c3e94a31bc8cb37c9de49884f9fbde25605bdb)